spec/services/variant_units/unit_prices_spec.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+module VariantUnits
+  describe UnitPrices do
+    subject { UnitPrices.new(variant) }
+    let(:variant) { Spree::Variant.new }
+    let(:product) { double }
```

Comment:
We should rather use [instance_double](https://relishapp.com/rspec/rspec-mocks/docs/verifying-doubles/using-an-instance-double) so we don't diverge from `Spree::Product`'s actual interface.